libraries/recipe_utils.rb

```diff
@@ -102,8 +102,8 @@ def self.opposite_package_name(package_base_name)
 
   # Returns the installed package name based on platform and package base name.
   # Written because Windows is dumb
-  def self.installed_package_name(platform_family, package_base_name)
-    return package_base_name unless platform_family == 'windows'
+  def self.installed_package_name(platform_family, package_base_name, systemd_unit_file_name)
+    return package_base_name unless systemd_unit_file_name != '' unless platform_family == 'windows'
```

Comment:
what is this change for?  I don't think this is needed, but if it is then you shouldn't have two unless statements.  It would be `unless systemd_unit_file_name != '' || platform_family == 'windows'`.